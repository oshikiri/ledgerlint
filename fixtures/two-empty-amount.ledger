2020-03-26
  Expences:Household essentials  200 JPY
  Expences:Food
  Assets:Cash
