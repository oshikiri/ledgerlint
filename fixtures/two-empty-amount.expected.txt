fixtures/two-empty-amount.ledger:1 Transaction contains two or more empty amount
